artemis/Chart.yaml

```diff
@@ -15,6 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
+#TODO: figure out how and when this is updated
```

Comment:
I don't know the process of how we update chart versions or if we do this at all for the `develop` branch?